api/access/invoker.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package access
+
+import (
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type Invoker interface {
+	GetAccount(address flow.Address, height uint64) (*flow.Account, error)
+	Script(height uint64, script []byte, parameters []cadence.Value) (cadence.Value, error)
```

Comment:
I think the `GetAccount` addition breaks the naming of the interface here. I want the `GetAccount` to be just called `Account` and have a more meaningful interface name (`Chain`, `State`, `View`?).